libminifi/include/capi/Plan.h

```diff
@@ -46,11 +48,50 @@
 #include "capi/cstructs.h"
 #include "capi/api.h"
 
+enum FailureStrategy { AS_IS, ROLLBACK };
```

Comment:
Do you think AS_IS could be clarified in the scope of the enumeration to better indicate the intent?

Reply:
I agree with you that it's not the best naming as doesn't properly describe what happens, but no better idea. @apiri - you are our last hope. :)